.gitpod.yml

```diff
@@ -1,35 +1,35 @@
 image:
   file: .gitpod.Dockerfile
 
-
 ports:
-    - name: Grafana
-      port: 3000
-      protocol: http
-      visibility: public
-      onOpen: ignore
-    - name: Health check
-      port: 8040
-      onOpen: ignore
-    - name: Lunar Proxy
-      port: 8080
-      onOpen: ignore
-    - name: Prometheus
-      port: 9090
-      onOpen: ignore
-    - name: Provider
-      port: 4578
-      onOpen: ignore
-    - name: Consumer
-      port: 7897
-      onOpen: ignore
+  - name: Grafana
+    port: 3000
+    protocol: http
+    visibility: public
+    onOpen: ignore
+  - name: Health check
+    port: 8040
+    onOpen: ignore
+  - name: Lunar Proxy
+    port: 8080
+    onOpen: ignore
+  - name: Prometheus
+    port: 9090
+    onOpen: ignore
+  - name: Provider
+    port: 4578
+    onOpen: ignore
+  - name: Consumer
+    port: 7897
+    onOpen: ignore
 
 tasks:
-    - name: Sandbox Terminal
-      command: |
-        open ${GITPOD_REPO_ROOT}/policies.yaml
-        cd /home/gitpod
-        POLICIES_DIR=${GITPOD_REPO_ROOT} docker compose --project-name sandbox up -d --wait --wait-timeout 60
-        for i in {1..3}; do curl --silent --output /dev/null -H "Content-Type: application/json" -d '{"scheme": "http", "host": "provider:4578", "path": "/rate-limit/exponential-backoff", "method": "GET", "sleep_ms": 20}' http://localhost:7897/traffic; done
-        gp ports await 3000
-        gp preview $(gp url 3000)/d/d2c76c83-9968-4ef6-a6a5-6105e661f9bf/lunar-sandbox
+  - name: Sandbox Terminal
+    command: |
+      cp ${GITPOD_REPO_ROOT}/scenarios/${SANDBOX_SCENARIO} ${GITPOD_REPO_ROOT}/policies.yaml
```

Comment:
we currently don't have scenarios, we don't want to add Strategy-Based Throttling scenario?

Reply:
this is a scenario